build.cmd

```diff
@@ -94,7 +94,6 @@ set __BuildTypeDebug=0
 set __BuildTypeChecked=0
 set __BuildTypeRelease=0
 set __BuildStandaloneGC="-DFEATURE_STANDALONE_GC=0"
```

Comment:
Do you still need this? Why not build the clrgc.dll all the time?

Reply:
I guess I just haven't flipped the switch yet (I recall this comment: https://github.com/dotnet/coreclr/pull/10951#issuecomment-294044213). I'd be happy to do that now!

Reply:
Ah, I recall that `FEATURE_EVENT_TRACE` is currently disabled when building to accommodate a standalone GC. It really only needs to be disabled for `clrgc`. This is an oversight that I can probably fix pretty easily with this PR, I think - I'll give it a try.